cuda_ext.py

```diff
@@ -11,6 +11,35 @@
 library_dir = os.path.dirname(os.path.abspath(__file__))
 extension_name = "exllama_ext"
 
+# another kludge to get things compiling in Windows
+windows = os.name == "nt"
+if windows:
+    def find_msvc():
+        for msvc_dir in ["C:\\Program Files" + a + "\\Microsoft Visual Studio\\" + b + "\\" + c + "\\VC\Tools\\MSVC\\"
+            for b in ["2022", "2019", "2017"]
+            for a in ["", " (x86)"]
```

Comment:
You probably want to replace `["", " (x86)"]` with
`[os.environ["ProgramW6432"], os.environ["ProgramFiles(x86)"]]`
and drop `C:\\Program Files`, some people have moved this folder to other drives.

If anyone else is wondering, the non-kludge way to do this would be install the vswhere pypi package and have it do discovery via vswhere.exe. So this kludge is a reasonable way to avoid the extra dependency.

Reply:
Done.